PlatformUI/PlatformUI/Components/Input/PlatformInput.swift

```diff
@@ -80,16 +80,21 @@ private struct PlatformInputView: View {
             .truncationMode(model.truncateMode)
     }
     
-    private var header: Text? {
+    private var header: AnyView? {
         guard let headerText = model.label else { return nil }
-        return Text(headerText)
-            .themeColor(foreground: .textTertiary)
-            .themeFont(fontSize: .smaller)
+        return HStack(spacing: 4) {
+            Text(headerText)
+                .themeColor(foreground: .textTertiary)
+                .themeFont(fontSize: .smaller)
+            model.labelAccessory
+            Spacer()
+        }.wrappedInAnyView()
```

Comment:
this allows for the "USD" image here
![Screenshot 2024-04-04 at 1 06 41 PM](https://github.com/dydxprotocol/v4-native-ios/assets/149746839/ec2fc6dc-8e1c-41d4-9e51-baffd2b52281)